### DISCUSSION TITLE
posts/shell-br-enter/

### DISCUSSION THREAD

**Discussion by giscus** [zhyunk/zhyunk#5] [General]
# posts/shell-br-enter/

명령어가 길어질 때 가독성을 위한 줄 바꿈 방법

https://study.zhyun.kim/posts/shell-br-enter/

**Comment by zhyunk** [comment]
> powershell에서 --help 사용하는 방법이 굉장히 어색했는데  
> 따로 정리하기는 애매해서 이곳에 추가

## powerShell --help 사용하기

### 첫 시도시 update-help 설치
```powershell
update-help
```
"UI 문화권 {ko-KR}을(를) 사용하는~~" 이라는 빨간 글씨로 메세지가 나타난다면 한국어로 지원되지 않는 몇 가지 명령어 때문에 경고가 뜬 것임으로 무시해도 된다고 한다.

<br>

### 명령어 도움말 조회
```powershell
get-help 키워드
```

#### ex) mkdir 키워드에 대한 정보 조회
> ```powershell
> get-help mkdir
> ```
> 
> #### 응답 내용
> ```
> 이름
>      mkdir
> 개요                                                                                                                    
> 
> 구문
>     mkdir [-Path] <String[]> [-Value <Object>] [-Force] [-Credential <PSCredential>] [-WhatIf] [-Confirm] [-UseTransact
>     ion] [<CommonParameters>]
> 
>     mkdir [[-Path] <String[]>] -Name <String> [-Value <Object>] [-Force] [-Credential <PSCredential>] [-WhatIf] [-Confi
>     rm] [-UseTransaction] [<CommonParameters>]
> 
> 
> 설명
> 
> 
> 관련 링크
> 
> 설명
>     예를 보려면 다음과 같이 입력하십시오. "get-help mkdir -examples".
>     자세한 내용을 보려면 다음과 같이 입력하십시오. "get-help mkdir -detailed".
>     기술적인 내용을 보려면 다음과 같이 입력하십시오. "get-help mkdir -full".
> ```
